### DISCUSSION TITLE
flomo-qq-bot:通过QQ对话将文字和图片保存到flomo

### DISCUSSION THREAD

**Discussion by iskycc** [flomoapp/3rd-party-tools#55] [分享创造]
通过QQ对话将文字和图片保存到flomo(需自行部署)

github: https://github.com/iskycc/flomo-qq-bot
gitee: https://gitee.com/iskycc/flomo-qq-bot

## 已实现的功能

①绑定
![](https://cdn.jsdelivr.net/gh/iskycc/flomo-qq-bot/pic/bb.jpg)
②保存文字或图片
![](https://cdn.jsdelivr.net/gh/iskycc/flomo-qq-bot/pic/aa.jpg)
③同时保存文字和图片
![](https://cdn.jsdelivr.net/gh/iskycc/flomo-qq-bot/pic/cc.jpg)
④解绑
![](https://cdn.jsdelivr.net/gh/iskycc/flomo-qq-bot/pic/dd.jpg)

**Comment by harry-devhub** [comment]
这个工具解决了官方微信无法发送图+文的内容的缺陷。很好的满足了我的需求，感谢大佬！
![图片](https://user-images.githubusercontent.com/46886133/128984306-7795ebfd-fb66-428e-8917-8b58f4a6bf52.png)

**Comment by cuminlo** [comment]
@iskycc 老哥，图片是怎么上传的？

**Comment by So-For** [comment]
大佬，王者带带我
![0BC3193F](https://user-images.githubusercontent.com/66558137/172814559-e2167608-82e9-4e5d-871b-346f7200adc1.png)
